dex/msgjson/types.go

```diff
@@ -789,25 +829,25 @@ type NotifyFeeResult struct {
 	signable
 }
 
-// ConfigResult is the successful result from the 'config' route.
-type ConfigResult struct {
-	CancelMax        float64  `json:"cancelmax"`
-	BroadcastTimeout uint64   `json:"btimeout"`
-	RegFeeConfirms   uint16   `json:"regfeeconfirms"`
-	Assets           []Asset  `json:"assets"`
-	Markets          []Market `json:"markets"`
-	Fee              uint64   `json:"fee"`
+// MarketStatus describes the status of the market, where StartEpoch is when the
+// market started or will start. FinalEpoch is a when the market will suspend
+// if it is running, or when the market suspended if it is presently stopped.
+type MarketStatus struct {
+	StartEpoch uint64 `json:"startepoch"`
+	FinalEpoch uint64 `json:"finalepoch,omitempty"`
+	Persist    *bool  `json:"persistbook,omitempty"` // nil and omitted when finalepoch is omitted
```

Comment:
I initially had a field `ActiveEpoch uint64 `json:"epoch"`, but due to the extra subsystem communication work I described this is not implemented.